Microcontroller_L432KC/README.md

```diff
@@ -0,0 +1,18 @@
+# ECE3SAT_PhaseB_EDT
+
+This code has been wrote for STM32 Nucleo L432KC.
+
+## Prerequisites
+
+You need to have an installed and configured [MbedCLI](https://os.mbed.com/docs/v5.7/tools/arm-mbed-cli.html).
+
+
+## Installation
+
+Use `mbed new .` in this directory to create a Mbed project with all configurations and dependencies in this repository.

```

Comment:
If this is intended to become a guide for a larger audience (for any project using a similar board), we should specify which directory and repository you are talking about...
So, either be more specific (so that one knows precisely what this guide is about, and can adapt each step to its own needs) or be more general (so that one has a general guide to how a project environment is set up).